fastparquet/writer.py

```diff
@@ -762,22 +764,44 @@ def make_metadata(data, has_nulls=True, ignore_columns=None, fixed_text=None,
 
 
 def write_simple(fn, data, fmd, row_group_offsets, compression,
-                 open_with, has_nulls, append=False, stats=True):
```

Comment:
Changing this signature, even if the argument is not used, might mess up other libraries that call it.

Reply:
Ok, I have added it back.
It was raising no error in pandas or dask test cases.
For my information, what should be the steps to operate such a change? (I guess at some point, cleaning a function signature makes sense to maintain a library?)

Reply:
Ideally, we would set it to `=None` and raise PendingDeprecationWarning if it has a non-None value.

Reply:
`PendingDeprecationWarning` is set for
- `has_nulls` in `writer.write_simple()`,
- `sep` in `api.ParquetFile.__init__()`

Reply:
Maybe remove these just for the moment, make this a separate PR (and perhaps after the next release)